package.json

```diff
@@ -868,6 +868,43 @@
           "description": "Specify GOPATH here to override the one that is set as environment variable. The inferred GOPATH from workspace root overrides this, if go.inferGopath is set to true.",
           "scope": "resource"
         },
+        "go.operationTimeout": {
```

Comment:
does this configuration have to be exposed to the user? I could see users caring about on-save timeouts, but I'm not sure about on-hover, etc.

Reply:
If these operations converge on the language server maybe it's less necessary than before 

This decoupling was originally conceived on the premise that actions are handled by different tools with different stability and performance guarantees (like guru, which triggered this work due to stagnating processes eating cpu)

Reply:
That's a good point @stamblerre 
One of my reasons of decoupling is that using the same value for onType and onHover might have unreasonable affects if I chose a larger value for the sake of a command.

But, if we take the approach where the setting only applies to commands & onSave operations, use a decent default for onType and onHover, we should be good.